addons/mail/models/mail_thread.py

```diff
@@ -4017,7 +4017,8 @@ def _message_update_content(self, message, body, attachment_ids=None,
         if strict:
             self._check_can_update_message_content(message.sudo())
 
-        msg_values = {'body': body} if body is not None else {}
+        msg_body = tools.html2plaintext(message.body)
```

Comment:
Why would we do that ?

Reply:
Same question here, what is the reason behind this change?